torch/csrc/stable/tensor_struct.h

```diff
@@ -189,6 +189,12 @@ class Tensor {
     return defined;
   }
 
+  Tensor clone() const {
```

Comment:
Is `clone` a method on at::Tensor/at::TensorBase? If not I think the guiding principle (https://github.com/pytorch/pytorch/pull/155367#issuecomment-2960051820) is that we want this one to live in ops.h

Reply:
`clone` is now a ops.h function.